2015/03/02 21:35:35 [error] [system.db.CDbCommand] Не удалось выполнить CDbCommand::fetchAll(): SQLSTATE[42S02]: Base table or view not found: 1146 Table 'db_yii.ml_content' doesn't exist. Выполнявшийся SQL-запрос: SHOW FULL COLUMNS FROM `ml_content`.
in E:\OpenServer\domains\Yii\index.php (13)
2015/03/02 22:26:50 [error] [exception.CException] exception 'CException' with message 'Контроллер UsersController не может найти представление "index".' in E:\OpenServer\domains\Yii\framework\web\CController.php:878
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\web\CController.php(782): CController->renderPartial('index', NULL, true)
#1 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CController->render('index')
#2 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionUser()
#3 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('user')
#7 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users/user')
#8 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#9 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=users/user
---
2015/03/02 22:27:28 [error] [exception.CException] exception 'CException' with message 'Контроллер UsersController не может найти представление "index".' in E:\OpenServer\domains\Yii\framework\web\CController.php:878
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\web\CController.php(782): CController->renderPartial('index', NULL, true)
#1 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CController->render('index')
#2 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionUser()
#3 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('user')
#7 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users/user')
#8 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#9 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=users/user
---
2015/03/02 22:28:00 [error] [exception.CException] exception 'CException' with message 'Контроллер UsersController не может найти представление "site/index".' in E:\OpenServer\domains\Yii\framework\web\CController.php:878
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\web\CController.php(782): CController->renderPartial('site/index', NULL, true)
#1 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CController->render('site/index')
#2 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionUser()
#3 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('user')
#7 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users/user')
#8 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#9 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=users/user
---
2015/03/02 22:28:37 [error] [exception.CException] exception 'CException' with message 'Контроллер UsersController не может найти представление "views/site/index".' in E:\OpenServer\domains\Yii\framework\web\CController.php:878
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\web\CController.php(782): CController->renderPartial('views/site/inde...', NULL, true)
#1 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CController->render('views/site/inde...')
#2 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionUser()
#3 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('user')
#7 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users/user')
#8 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#9 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=users/user
---
2015/03/02 22:28:48 [error] [exception.CException] exception 'CException' with message 'Контроллер UsersController не может найти представление "index".' in E:\OpenServer\domains\Yii\framework\web\CController.php:878
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\web\CController.php(782): CController->renderPartial('index', NULL, true)
#1 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CController->render('index')
#2 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionUser()
#3 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('user')
#7 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users/user')
#8 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#9 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=users/user
---
2015/03/02 22:28:49 [error] [exception.CException] exception 'CException' with message 'Контроллер UsersController не может найти представление "index".' in E:\OpenServer\domains\Yii\framework\web\CController.php:878
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\web\CController.php(782): CController->renderPartial('index', NULL, true)
#1 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CController->render('index')
#2 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionUser()
#3 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('user')
#7 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users/user')
#8 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#9 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=users/user
---
2015/03/02 22:57:40 [error] [system.db.CDbCommand] Не удалось выполнить CDbCommand::fetchAll(): SQLSTATE[42S02]: Base table or view not found: 1146 Table 'db_yii.user' doesn't exist. Выполнявшийся SQL-запрос: SHOW FULL COLUMNS FROM `user`.
in E:\OpenServer\domains\Yii\index.php (13)
2015/03/02 23:01:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "user".' in E:\OpenServer\domains\Yii\framework\web\CWebApplication.php:286
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#1 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=user
---
2015/03/02 23:03:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "user".' in E:\OpenServer\domains\Yii\framework\web\CWebApplication.php:286
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#1 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=user
---
2015/03/02 23:03:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "user".' in E:\OpenServer\domains\Yii\framework\web\CWebApplication.php:286
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#1 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=user
---
2015/03/02 23:13:38 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "User.title".' in E:\OpenServer\domains\Yii\framework\base\CComponent.php:130
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('title')
#1 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(8): CActiveRecord->__get('title')
#2 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#3 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#7 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#8 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#9 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/02 23:23:42 [error] [system.db.CDbCommand] Не удалось выполнить CDbCommand::fetch(): SQLSTATE[HY093]: Invalid parameter number: no parameters were bound. Выполнявшийся SQL-запрос: SELECT * FROM `ml_user` `t` WHERE id<:num LIMIT 1.
in E:\OpenServer\domains\Yii\protected\controllers\UsersController.php (7)
in E:\OpenServer\domains\Yii\index.php (13)
2015/03/02 23:23:42 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand не удалось исполнить SQL-запрос: SQLSTATE[HY093]: Invalid parameter number: no parameters were bound. The SQL statement executed was: SELECT * FROM `ml_user` `t` WHERE id<:num LIMIT 1' in E:\OpenServer\domains\Yii\framework\db\CDbCommand.php:543
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\CDbCommand.php(415): CDbCommand->queryInternal('fetch', Array, Array)
#1 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1356): CDbCommand->queryRow()
#2 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1461): CActiveRecord->query(Object(CDbCriteria))
#3 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CActiveRecord->find('id<:num')
#4 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#9 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#10 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 17:20:04 [error] [php] Invalid argument supplied for foreach() (E:\OpenServer\domains\Yii\protected\controllers\UsersController.php:8)
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\web\CController.php(286): UsersController->runAction()
#1 E:\OpenServer\domains\Yii\framework\web\CController.php(265): UsersController->runActionWithFilters()
#2 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): UsersController->run()
#3 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#5 E:\OpenServer\domains\Yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=users
in E:\OpenServer\domains\Yii\protected\controllers\UsersController.php (8)
in E:\OpenServer\domains\Yii\index.php (13)
2015/03/03 17:23:08 [error] [system.db.CDbCommand] Не удалось выполнить CDbCommand::fetchAll(): SQLSTATE[HY093]: Invalid parameter number: parameter was not defined. Выполнявшийся SQL-запрос: SELECT * FROM `ml_user` `t` WHERE id=:ids.
in E:\OpenServer\domains\Yii\protected\controllers\UsersController.php (7)
in E:\OpenServer\domains\Yii\index.php (13)
2015/03/03 17:23:08 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand не удалось исполнить SQL-запрос: SQLSTATE[HY093]: Invalid parameter number: parameter was not defined. The SQL statement executed was: SELECT * FROM `ml_user` `t` WHERE id=:ids' in E:\OpenServer\domains\Yii\framework\db\CDbCommand.php:543
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1356): CDbCommand->queryAll()
#2 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1475): CActiveRecord->query(Object(CDbCriteria), true)
#3 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CActiveRecord->findAll('id=:ids', Array)
#4 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#9 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#10 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 17:24:35 [error] [system.db.CDbCommand] Не удалось выполнить CDbCommand::fetchAll(): SQLSTATE[HY093]: Invalid parameter number: parameter was not defined. Выполнявшийся SQL-запрос: SELECT * FROM `ml_user` `t` WHERE id=:ids.
in E:\OpenServer\domains\Yii\protected\controllers\UsersController.php (7)
in E:\OpenServer\domains\Yii\index.php (13)
2015/03/03 17:24:35 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand не удалось исполнить SQL-запрос: SQLSTATE[HY093]: Invalid parameter number: parameter was not defined. The SQL statement executed was: SELECT * FROM `ml_user` `t` WHERE id=:ids' in E:\OpenServer\domains\Yii\framework\db\CDbCommand.php:543
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1356): CDbCommand->queryAll()
#2 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1475): CActiveRecord->query(Object(CDbCriteria), true)
#3 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CActiveRecord->findAll('id=:ids', Array)
#4 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#9 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#10 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 17:26:07 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CDbCriteria.id".' in E:\OpenServer\domains\Yii\framework\base\CComponent.php:173
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\schema\CDbCriteria.php(160): CComponent->__set('id', Array)
#1 E:\OpenServer\domains\Yii\framework\db\schema\CDbCommandBuilder.php(588): CDbCriteria->__construct(Array)
#2 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1474): CDbCommandBuilder->createCriteria(Array, Array)
#3 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CActiveRecord->findAll(Array, Array)
#4 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#9 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#10 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 17:58:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "user".' in E:\OpenServer\domains\Yii\framework\web\CWebApplication.php:286
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#1 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=user
---
2015/03/03 17:58:25 [error] [php] Object of class Cat could not be converted to string (E:\OpenServer\domains\Yii\protected\controllers\UsersController.php:7)
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\web\CController.php(286): UsersController->runAction()
#1 E:\OpenServer\domains\Yii\framework\web\CController.php(265): UsersController->runActionWithFilters()
#2 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): UsersController->run()
#3 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#5 E:\OpenServer\domains\Yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=users
in E:\OpenServer\domains\Yii\protected\controllers\UsersController.php (7)
in E:\OpenServer\domains\Yii\index.php (13)
2015/03/03 17:59:13 [error] [php] Object of class Cat could not be converted to string (E:\OpenServer\domains\Yii\protected\controllers\UsersController.php:7)
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\web\CController.php(286): UsersController->runAction()
#1 E:\OpenServer\domains\Yii\framework\web\CController.php(265): UsersController->runActionWithFilters()
#2 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): UsersController->run()
#3 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#5 E:\OpenServer\domains\Yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=users
in E:\OpenServer\domains\Yii\protected\controllers\UsersController.php (7)
in E:\OpenServer\domains\Yii\index.php (13)
2015/03/03 18:09:35 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CDbCriteria.0".' in E:\OpenServer\domains\Yii\framework\base\CComponent.php:173
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\schema\CDbCriteria.php(160): CComponent->__set('0', 'id<:num')
#1 E:\OpenServer\domains\Yii\framework\db\schema\CDbCommandBuilder.php(588): CDbCriteria->__construct(Array)
#2 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1460): CDbCommandBuilder->createCriteria(Array, Array)
#3 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(6): CActiveRecord->find(Array, Array)
#4 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#9 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#10 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 18:10:32 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CDbCriteria.0".' in E:\OpenServer\domains\Yii\framework\base\CComponent.php:173
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\schema\CDbCriteria.php(160): CComponent->__set('0', 'id<:id')
#1 E:\OpenServer\domains\Yii\framework\db\schema\CDbCommandBuilder.php(588): CDbCriteria->__construct(Array)
#2 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1460): CDbCommandBuilder->createCriteria(Array, Array)
#3 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(6): CActiveRecord->find(Array, Array)
#4 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#9 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#10 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 18:10:51 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CDbCriteria.0".' in E:\OpenServer\domains\Yii\framework\base\CComponent.php:173
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\schema\CDbCriteria.php(160): CComponent->__set('0', 'id=:id')
#1 E:\OpenServer\domains\Yii\framework\db\schema\CDbCommandBuilder.php(588): CDbCriteria->__construct(Array)
#2 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1460): CDbCommandBuilder->createCriteria(Array, Array)
#3 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(6): CActiveRecord->find(Array, Array)
#4 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#9 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#10 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 18:14:13 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CDbCriteria.0".' in E:\OpenServer\domains\Yii\framework\base\CComponent.php:173
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\schema\CDbCriteria.php(160): CComponent->__set('0', 'name=:name')
#1 E:\OpenServer\domains\Yii\framework\db\schema\CDbCommandBuilder.php(588): CDbCriteria->__construct(Array)
#2 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1474): CDbCommandBuilder->createCriteria(Array, Array)
#3 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CActiveRecord->findAll(Array, Array)
#4 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#9 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#10 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 18:14:18 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CDbCriteria.0".' in E:\OpenServer\domains\Yii\framework\base\CComponent.php:173
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\schema\CDbCriteria.php(160): CComponent->__set('0', 'name=:name')
#1 E:\OpenServer\domains\Yii\framework\db\schema\CDbCommandBuilder.php(588): CDbCriteria->__construct(Array)
#2 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1474): CDbCommandBuilder->createCriteria(Array, Array)
#3 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CActiveRecord->findAll(Array, Array)
#4 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#9 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#10 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 18:14:54 [error] [system.db.CDbCommand] Не удалось выполнить CDbCommand::fetchAll(): SQLSTATE[HY093]: Invalid parameter number: parameter was not defined. Выполнявшийся SQL-запрос: SELECT * FROM `ml_cat` `t` WHERE name=:name.
in E:\OpenServer\domains\Yii\protected\controllers\UsersController.php (7)
in E:\OpenServer\domains\Yii\index.php (13)
2015/03/03 18:14:54 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand не удалось исполнить SQL-запрос: SQLSTATE[HY093]: Invalid parameter number: parameter was not defined. The SQL statement executed was: SELECT * FROM `ml_cat` `t` WHERE name=:name' in E:\OpenServer\domains\Yii\framework\db\CDbCommand.php:543
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1356): CDbCommand->queryAll()
#2 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1475): CActiveRecord->query(Object(CDbCriteria), true)
#3 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CActiveRecord->findAll('name=:name', 'id=:id', Array)
#4 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#9 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#10 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 18:15:02 [error] [system.db.CDbCommand] Не удалось выполнить CDbCommand::fetchAll(): SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens. Выполнявшийся SQL-запрос: SELECT * FROM `ml_cat` `t` WHERE name=:name.
in E:\OpenServer\domains\Yii\protected\controllers\UsersController.php (7)
in E:\OpenServer\domains\Yii\index.php (13)
2015/03/03 18:15:02 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand не удалось исполнить SQL-запрос: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens. The SQL statement executed was: SELECT * FROM `ml_cat` `t` WHERE name=:name' in E:\OpenServer\domains\Yii\framework\db\CDbCommand.php:543
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1356): CDbCommand->queryAll()
#2 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1475): CActiveRecord->query(Object(CDbCriteria), true)
#3 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CActiveRecord->findAll('name=:name', Array)
#4 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#9 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#10 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 18:18:05 [error] [php] include(Users.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory (E:\OpenServer\domains\Yii\framework\YiiBase.php:432)
Stack trace:
#0 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): spl_autoload_call()
#1 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#2 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#3 E:\OpenServer\domains\Yii\framework\web\CController.php(286): UsersController->runAction()
#4 E:\OpenServer\domains\Yii\framework\web\CController.php(265): UsersController->runActionWithFilters()
#5 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): UsersController->run()
#6 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#7 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#8 E:\OpenServer\domains\Yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=users
in E:\OpenServer\domains\Yii\protected\controllers\UsersController.php (7)
in E:\OpenServer\domains\Yii\index.php (13)
2015/03/03 18:27:57 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CDbCriteria.name".' in E:\OpenServer\domains\Yii\framework\base\CComponent.php:173
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\schema\CDbCriteria.php(160): CComponent->__set('name', 'krazydragod')
#1 E:\OpenServer\domains\Yii\framework\db\schema\CDbCommandBuilder.php(588): CDbCriteria->__construct(Array)
#2 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1668): CDbCommandBuilder->createCriteria(Array, Array)
#3 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CActiveRecord->exists(Array)
#4 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#9 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#10 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 18:28:40 [error] [php] Invalid argument supplied for foreach() (E:\OpenServer\domains\Yii\protected\controllers\UsersController.php:8)
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\web\CController.php(286): UsersController->runAction()
#1 E:\OpenServer\domains\Yii\framework\web\CController.php(265): UsersController->runActionWithFilters()
#2 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): UsersController->run()
#3 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#5 E:\OpenServer\domains\Yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=users
in E:\OpenServer\domains\Yii\protected\controllers\UsersController.php (8)
in E:\OpenServer\domains\Yii\index.php (13)
2015/03/03 18:32:35 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CDbCriteria.name".' in E:\OpenServer\domains\Yii\framework\base\CComponent.php:173
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\schema\CDbCriteria.php(160): CComponent->__set('name', ':name')
#1 E:\OpenServer\domains\Yii\framework\db\schema\CDbCommandBuilder.php(588): CDbCriteria->__construct(Array)
#2 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1602): CDbCommandBuilder->createCriteria(Array, Array)
#3 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(9): CActiveRecord->count(Array)
#4 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#9 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#10 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 18:34:48 [error] [system.db.CDbCommand] Не удалось выполнить CDbCommand::fetch(): SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'array(':name'=>krazydragod) LIMIT 1' at line 1. Выполнявшийся SQL-запрос: SELECT 1 FROM `ml_user` `t` WHERE 'name=:name',array(':name'=>krazydragod) LIMIT 1.
in E:\OpenServer\domains\Yii\protected\controllers\UsersController.php (8)
in E:\OpenServer\domains\Yii\index.php (13)
2015/03/03 18:34:48 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand не удалось исполнить SQL-запрос: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'array(':name'=>krazydragod) LIMIT 1' at line 1. The SQL statement executed was: SELECT 1 FROM `ml_user` `t` WHERE 'name=:name',array(':name'=>krazydragod) LIMIT 1' in E:\OpenServer\domains\Yii\framework\db\CDbCommand.php:543
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\CDbCommand.php(415): CDbCommand->queryInternal('fetch', Array, Array)
#1 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1675): CDbCommand->queryRow()
#2 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(8): CActiveRecord->exists(''name=:name',ar...')
#3 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#4 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#9 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 18:56:50 [error] [exception.CException] exception 'CException' with message 'В классе User и его поведениях не найден метод или замыкание с именем "FindAttributes".' in E:\OpenServer\domains\Yii\framework\base\CComponent.php:266
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(227): CComponent->__call('FindAttributes', Array)
#1 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CActiveRecord->__call('FindAttributes', Array)
#2 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): User->FindAttributes('name=:name', Array)
#3 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#4 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#9 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 18:57:12 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CDbCriteria.:name".' in E:\OpenServer\domains\Yii\framework\base\CComponent.php:173
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\schema\CDbCriteria.php(160): CComponent->__set(':name', 'krazydragod2')
#1 E:\OpenServer\domains\Yii\framework\db\schema\CDbCommandBuilder.php(588): CDbCriteria->__construct(Array)
#2 E:\OpenServer\domains\Yii\framework\db\schema\CDbCommandBuilder.php(664): CDbCommandBuilder->createCriteria(Array, Array)
#3 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1523): CDbCommandBuilder->createColumnCriteria(Object(CMysqlTableSchema), 'name=:name', Array, Array, '`t`.')
#4 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CActiveRecord->findByAttributes('name=:name', Array)
#5 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#10 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#11 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#12 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 18:57:31 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CDbCriteria.:name".' in E:\OpenServer\domains\Yii\framework\base\CComponent.php:173
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\schema\CDbCriteria.php(160): CComponent->__set(':name', 'krazydragod')
#1 E:\OpenServer\domains\Yii\framework\db\schema\CDbCommandBuilder.php(588): CDbCriteria->__construct(Array)
#2 E:\OpenServer\domains\Yii\framework\db\schema\CDbCommandBuilder.php(664): CDbCommandBuilder->createCriteria(Array, Array)
#3 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1523): CDbCommandBuilder->createColumnCriteria(Object(CMysqlTableSchema), 'name=:name', Array, Array, '`t`.')
#4 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CActiveRecord->findByAttributes('name=:name', Array)
#5 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#10 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#11 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#12 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 18:58:38 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CDbCriteria.:name".' in E:\OpenServer\domains\Yii\framework\base\CComponent.php:173
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\schema\CDbCriteria.php(160): CComponent->__set(':name', 'krazydragod')
#1 E:\OpenServer\domains\Yii\framework\db\schema\CDbCommandBuilder.php(588): CDbCriteria->__construct(Array)
#2 E:\OpenServer\domains\Yii\framework\db\schema\CDbCommandBuilder.php(664): CDbCommandBuilder->createCriteria(Array, Array)
#3 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1523): CDbCommandBuilder->createColumnCriteria(Object(CMysqlTableSchema), Array, Array, Array, '`t`.')
#4 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CActiveRecord->findByAttributes(Array, Array)
#5 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#10 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#11 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#12 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 19:00:10 [error] [php] Invalid argument supplied for foreach() (E:\OpenServer\domains\Yii\protected\controllers\UsersController.php:11)
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\web\CController.php(286): UsersController->runAction()
#1 E:\OpenServer\domains\Yii\framework\web\CController.php(265): UsersController->runActionWithFilters()
#2 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): UsersController->run()
#3 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#5 E:\OpenServer\domains\Yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=users
in E:\OpenServer\domains\Yii\protected\controllers\UsersController.php (11)
in E:\OpenServer\domains\Yii\index.php (13)
2015/03/03 19:00:15 [error] [php] Invalid argument supplied for foreach() (E:\OpenServer\domains\Yii\protected\controllers\UsersController.php:11)
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\web\CController.php(286): UsersController->runAction()
#1 E:\OpenServer\domains\Yii\framework\web\CController.php(265): UsersController->runActionWithFilters()
#2 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): UsersController->run()
#3 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#5 E:\OpenServer\domains\Yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=users
in E:\OpenServer\domains\Yii\protected\controllers\UsersController.php (11)
in E:\OpenServer\domains\Yii\index.php (13)
2015/03/03 19:00:23 [error] [php] Invalid argument supplied for foreach() (E:\OpenServer\domains\Yii\protected\controllers\UsersController.php:11)
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\web\CController.php(286): UsersController->runAction()
#1 E:\OpenServer\domains\Yii\framework\web\CController.php(265): UsersController->runActionWithFilters()
#2 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): UsersController->run()
#3 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#5 E:\OpenServer\domains\Yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=users
in E:\OpenServer\domains\Yii\protected\controllers\UsersController.php (11)
in E:\OpenServer\domains\Yii\index.php (13)
2015/03/03 19:01:19 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CDbCriteria.:name".' in E:\OpenServer\domains\Yii\framework\base\CComponent.php:173
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\schema\CDbCriteria.php(160): CComponent->__set(':name', 'krazydragod22')
#1 E:\OpenServer\domains\Yii\framework\db\schema\CDbCommandBuilder.php(588): CDbCriteria->__construct(Array)
#2 E:\OpenServer\domains\Yii\framework\db\schema\CDbCommandBuilder.php(664): CDbCommandBuilder->createCriteria(Array, Array)
#3 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1523): CDbCommandBuilder->createColumnCriteria(Object(CMysqlTableSchema), Array, Array, Array, '`t`.')
#4 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CActiveRecord->findByAttributes(Array, Array)
#5 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#10 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#11 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#12 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 19:08:52 [error] [exception.CDbException] exception 'CDbException' with message 'Таблица "ml_user" не содержит столбца "0".' in E:\OpenServer\domains\Yii\framework\db\schema\CDbCommandBuilder.php:697
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1523): CDbCommandBuilder->createColumnCriteria(Object(CMysqlTableSchema), Array, '', Array, '`t`.')
#1 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CActiveRecord->findByAttributes(Array)
#2 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#3 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#7 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#8 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#9 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 19:20:53 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CDbCriteria.name".' in E:\OpenServer\domains\Yii\framework\base\CComponent.php:173
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\schema\CDbCriteria.php(160): CComponent->__set('name', 'krazydragod22')
#1 E:\OpenServer\domains\Yii\framework\db\schema\CDbCommandBuilder.php(588): CDbCriteria->__construct(Array)
#2 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1602): CDbCommandBuilder->createCriteria(Array, Array)
#3 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CActiveRecord->count(Array)
#4 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#9 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#10 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 19:22:41 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CDbCriteria.name".' in E:\OpenServer\domains\Yii\framework\base\CComponent.php:173
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\schema\CDbCriteria.php(160): CComponent->__set('name', ':name')
#1 E:\OpenServer\domains\Yii\framework\db\schema\CDbCommandBuilder.php(588): CDbCriteria->__construct(Array)
#2 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1602): CDbCommandBuilder->createCriteria(Array, Array)
#3 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CActiveRecord->count(Array, Array)
#4 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#9 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#10 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 19:23:15 [error] [system.db.CDbCommand] Не удалось выполнить CDbCommand::fetchColumn(): SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ' id=3' at line 1. Выполнявшийся SQL-запрос: SELECT COUNT(*) FROM `ml_user` `t` WHERE name=:name, id=:id.
in E:\OpenServer\domains\Yii\protected\controllers\UsersController.php (7)
in E:\OpenServer\domains\Yii\index.php (13)
2015/03/03 19:23:15 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand не удалось исполнить SQL-запрос: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ' id=3' at line 1. The SQL statement executed was: SELECT COUNT(*) FROM `ml_user` `t` WHERE name=:name, id=:id' in E:\OpenServer\domains\Yii\framework\db\CDbCommand.php:543
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\CDbCommand.php(433): CDbCommand->queryInternal('fetchColumn', 0, Array)
#1 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1606): CDbCommand->queryScalar()
#2 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CActiveRecord->count('name=:name, id=...', Array)
#3 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#4 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#9 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 19:23:36 [error] [system.db.CDbCommand] Не удалось выполнить CDbCommand::fetchColumn(): SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens. Выполнявшийся SQL-запрос: SELECT COUNT(*) FROM `ml_user` `t` WHERE name=:name.
in E:\OpenServer\domains\Yii\protected\controllers\UsersController.php (7)
in E:\OpenServer\domains\Yii\index.php (13)
2015/03/03 19:23:36 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand не удалось исполнить SQL-запрос: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens. The SQL statement executed was: SELECT COUNT(*) FROM `ml_user` `t` WHERE name=:name' in E:\OpenServer\domains\Yii\framework\db\CDbCommand.php:543
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\CDbCommand.php(433): CDbCommand->queryInternal('fetchColumn', 0, Array)
#1 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1606): CDbCommand->queryScalar()
#2 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CActiveRecord->count('name=:name', Array)
#3 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#4 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#9 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 19:27:03 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CDbCriteria.name".' in E:\OpenServer\domains\Yii\framework\base\CComponent.php:173
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\schema\CDbCriteria.php(160): CComponent->__set('name', 'krazydragod222')
#1 E:\OpenServer\domains\Yii\framework\db\schema\CDbCommandBuilder.php(588): CDbCriteria->__construct(Array)
#2 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1769): CDbCommandBuilder->createCriteria(Array, Array)
#3 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CActiveRecord->updateAll('name="krazydrag...', Array)
#4 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#9 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#10 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 19:28:55 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CDbCriteria.name".' in E:\OpenServer\domains\Yii\framework\base\CComponent.php:173
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\schema\CDbCriteria.php(160): CComponent->__set('name', 'krazydragod222')
#1 E:\OpenServer\domains\Yii\framework\db\schema\CDbCommandBuilder.php(588): CDbCriteria->__construct(Array)
#2 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1769): CDbCommandBuilder->createCriteria(Array, Array)
#3 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CActiveRecord->updateAll(Array, Array)
#4 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#9 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#10 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 19:40:17 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CDbCriteria.name".' in E:\OpenServer\domains\Yii\framework\base\CComponent.php:173
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\db\schema\CDbCriteria.php(160): CComponent->__set('name', 'krazydragod22')
#1 E:\OpenServer\domains\Yii\framework\db\schema\CDbCommandBuilder.php(588): CDbCriteria->__construct(Array)
#2 E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php(1821): CDbCommandBuilder->createCriteria(Array, Array)
#3 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(7): CActiveRecord->deleteAll(Array)
#4 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 E:\OpenServer\domains\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): CController->run('')
#9 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#10 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 E:\OpenServer\domains\Yii\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=users
---
2015/03/03 19:51:49 [error] [php] Missing argument 2 for CActiveRecord::updateByPk(), called in E:\OpenServer\domains\Yii\protected\controllers\UsersController.php on line 10 and defined (E:\OpenServer\domains\Yii\framework\db\ar\CActiveRecord.php:1746)
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#1 E:\OpenServer\domains\Yii\framework\web\CController.php(286): UsersController->runAction()
#2 E:\OpenServer\domains\Yii\framework\web\CController.php(265): UsersController->runActionWithFilters()
#3 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): UsersController->run()
#4 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#5 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#6 E:\OpenServer\domains\Yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=users
in E:\OpenServer\domains\Yii\protected\controllers\UsersController.php (10)
in E:\OpenServer\domains\Yii\index.php (13)
2015/03/03 20:12:40 [error] [php] Invalid argument supplied for foreach() (E:\OpenServer\domains\Yii\protected\views\users\index.php:2)
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\web\CController.php(869): UsersController->renderFile()
#1 E:\OpenServer\domains\Yii\framework\web\CController.php(782): UsersController->renderPartial()
#2 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(14): UsersController->render()
#3 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#4 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(286): UsersController->runAction()
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(265): UsersController->runActionWithFilters()
#7 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): UsersController->run()
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 E:\OpenServer\domains\Yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=users
in E:\OpenServer\domains\Yii\protected\views\users\index.php (2)
in E:\OpenServer\domains\Yii\protected\controllers\UsersController.php (14)
in E:\OpenServer\domains\Yii\index.php (13)
2015/03/03 20:13:02 [error] [php] Invalid argument supplied for foreach() (E:\OpenServer\domains\Yii\protected\views\users\index.php:2)
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\web\CController.php(869): UsersController->renderFile()
#1 E:\OpenServer\domains\Yii\framework\web\CController.php(782): UsersController->renderPartial()
#2 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(14): UsersController->render()
#3 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#4 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(286): UsersController->runAction()
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(265): UsersController->runActionWithFilters()
#7 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): UsersController->run()
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 E:\OpenServer\domains\Yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=users
in E:\OpenServer\domains\Yii\protected\views\users\index.php (2)
in E:\OpenServer\domains\Yii\protected\controllers\UsersController.php (14)
in E:\OpenServer\domains\Yii\index.php (13)
2015/03/03 20:13:09 [error] [php] Invalid argument supplied for foreach() (E:\OpenServer\domains\Yii\protected\views\users\index.php:2)
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\web\CController.php(869): UsersController->renderFile()
#1 E:\OpenServer\domains\Yii\framework\web\CController.php(782): UsersController->renderPartial()
#2 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(14): UsersController->render()
#3 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#4 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(286): UsersController->runAction()
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(265): UsersController->runActionWithFilters()
#7 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): UsersController->run()
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 E:\OpenServer\domains\Yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=users
in E:\OpenServer\domains\Yii\protected\views\users\index.php (2)
in E:\OpenServer\domains\Yii\protected\controllers\UsersController.php (14)
in E:\OpenServer\domains\Yii\index.php (13)
2015/03/03 20:13:23 [error] [php] Invalid argument supplied for foreach() (E:\OpenServer\domains\Yii\protected\views\users\index.php:2)
Stack trace:
#0 E:\OpenServer\domains\Yii\framework\web\CController.php(869): UsersController->renderFile()
#1 E:\OpenServer\domains\Yii\framework\web\CController.php(782): UsersController->renderPartial()
#2 E:\OpenServer\domains\Yii\protected\controllers\UsersController.php(14): UsersController->render()
#3 E:\OpenServer\domains\Yii\framework\web\actions\CInlineAction.php(49): UsersController->actionIndex()
#4 E:\OpenServer\domains\Yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 E:\OpenServer\domains\Yii\framework\web\CController.php(286): UsersController->runAction()
#6 E:\OpenServer\domains\Yii\framework\web\CController.php(265): UsersController->runActionWithFilters()
#7 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(282): UsersController->run()
#8 E:\OpenServer\domains\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 E:\OpenServer\domains\Yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 E:\OpenServer\domains\Yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=users
in E:\OpenServer\domains\Yii\protected\views\users\index.php (2)
in E:\OpenServer\domains\Yii\protected\controllers\UsersController.php (14)
in E:\OpenServer\domains\Yii\index.php (13)
2015/04/28 21:08:31 [error] [php] date(): It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected the timezone 'UTC' for now, but please set date.timezone to select your timezone. (E:\OpenServer\domains\yii\protected\views\layouts\main.php:51)
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CBaseController.php(95): SiteController->renderInternal()
#1 E:\OpenServer\domains\yii\framework\web\widgets\CContentDecorator.php(76): SiteController->renderFile()
#2 E:\OpenServer\domains\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate()
#3 E:\OpenServer\domains\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput()
#4 E:\OpenServer\domains\yii\framework\web\CBaseController.php(214): CContentDecorator->run()
#5 E:\OpenServer\domains\yii\framework\web\CBaseController.php(308): SiteController->endWidget()
#6 E:\OpenServer\domains\yii\protected\views\layouts\column1.php(6): SiteController->endContent()
#7 E:\OpenServer\domains\yii\framework\web\CBaseController.php(126): require()
#8 E:\OpenServer\domains\yii\framework\web\CBaseController.php(95): SiteController->renderInternal()
#9 E:\OpenServer\domains\yii\framework\web\CController.php(784): SiteController->renderFile()
#10 E:\OpenServer\domains\yii\protected\controllers\SiteController.php(32): SiteController->render()
#11 E:\OpenServer\domains\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#12 E:\OpenServer\domains\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#13 E:\OpenServer\domains\yii\framework\web\CController.php(286): SiteController->runAction()
#14 E:\OpenServer\domains\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#15 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): SiteController->run()
#16 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#17 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#18 E:\OpenServer\domains\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/Yii/
in E:\OpenServer\domains\yii\protected\views\layouts\main.php (51)
in E:\OpenServer\domains\yii\protected\views\layouts\column1.php (6)
in E:\OpenServer\domains\yii\protected\controllers\SiteController.php (32)
2015/04/28 21:08:42 [error] [php] date(): It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected the timezone 'UTC' for now, but please set date.timezone to select your timezone. (E:\OpenServer\domains\yii\protected\views\layouts\main.php:51)
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CBaseController.php(95): SiteController->renderInternal()
#1 E:\OpenServer\domains\yii\framework\web\widgets\CContentDecorator.php(76): SiteController->renderFile()
#2 E:\OpenServer\domains\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate()
#3 E:\OpenServer\domains\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput()
#4 E:\OpenServer\domains\yii\framework\web\CBaseController.php(214): CContentDecorator->run()
#5 E:\OpenServer\domains\yii\framework\web\CBaseController.php(308): SiteController->endWidget()
#6 E:\OpenServer\domains\yii\protected\views\layouts\column1.php(6): SiteController->endContent()
#7 E:\OpenServer\domains\yii\framework\web\CBaseController.php(126): require()
#8 E:\OpenServer\domains\yii\framework\web\CBaseController.php(95): SiteController->renderInternal()
#9 E:\OpenServer\domains\yii\framework\web\CController.php(784): SiteController->renderFile()
#10 E:\OpenServer\domains\yii\protected\controllers\SiteController.php(32): SiteController->render()
#11 E:\OpenServer\domains\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#12 E:\OpenServer\domains\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#13 E:\OpenServer\domains\yii\framework\web\CController.php(286): SiteController->runAction()
#14 E:\OpenServer\domains\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#15 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): SiteController->run()
#16 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#17 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#18 E:\OpenServer\domains\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/Yii/
in E:\OpenServer\domains\yii\protected\views\layouts\main.php (51)
in E:\OpenServer\domains\yii\protected\views\layouts\column1.php (6)
in E:\OpenServer\domains\yii\protected\controllers\SiteController.php (32)
2015/04/28 21:08:46 [error] [php] date(): It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected the timezone 'UTC' for now, but please set date.timezone to select your timezone. (E:\OpenServer\domains\yii\protected\views\layouts\main.php:51)
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CBaseController.php(95): SiteController->renderInternal()
#1 E:\OpenServer\domains\yii\framework\web\widgets\CContentDecorator.php(76): SiteController->renderFile()
#2 E:\OpenServer\domains\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate()
#3 E:\OpenServer\domains\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput()
#4 E:\OpenServer\domains\yii\framework\web\CBaseController.php(214): CContentDecorator->run()
#5 E:\OpenServer\domains\yii\framework\web\CBaseController.php(308): SiteController->endWidget()
#6 E:\OpenServer\domains\yii\protected\views\layouts\column1.php(6): SiteController->endContent()
#7 E:\OpenServer\domains\yii\framework\web\CBaseController.php(126): require()
#8 E:\OpenServer\domains\yii\framework\web\CBaseController.php(95): SiteController->renderInternal()
#9 E:\OpenServer\domains\yii\framework\web\CController.php(784): SiteController->renderFile()
#10 E:\OpenServer\domains\yii\framework\web\actions\CViewAction.php(140): SiteController->render()
#11 E:\OpenServer\domains\yii\framework\web\actions\CAction.php(76): CViewAction->run()
#12 E:\OpenServer\domains\yii\framework\web\CController.php(308): CViewAction->runWithParams()
#13 E:\OpenServer\domains\yii\framework\web\CController.php(286): SiteController->runAction()
#14 E:\OpenServer\domains\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#15 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): SiteController->run()
#16 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#17 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#18 E:\OpenServer\domains\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/Yii//index.php?r=site/page&view=about
in E:\OpenServer\domains\yii\protected\views\layouts\main.php (51)
in E:\OpenServer\domains\yii\protected\views\layouts\column1.php (6)
in E:\OpenServer\domains\yii\index.php (13)
2015/04/28 21:08:48 [error] [php] date(): It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected the timezone 'UTC' for now, but please set date.timezone to select your timezone. (E:\OpenServer\domains\yii\protected\views\layouts\main.php:51)
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CBaseController.php(95): SiteController->renderInternal()
#1 E:\OpenServer\domains\yii\framework\web\widgets\CContentDecorator.php(76): SiteController->renderFile()
#2 E:\OpenServer\domains\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate()
#3 E:\OpenServer\domains\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput()
#4 E:\OpenServer\domains\yii\framework\web\CBaseController.php(214): CContentDecorator->run()
#5 E:\OpenServer\domains\yii\framework\web\CBaseController.php(308): SiteController->endWidget()
#6 E:\OpenServer\domains\yii\protected\views\layouts\column1.php(6): SiteController->endContent()
#7 E:\OpenServer\domains\yii\framework\web\CBaseController.php(126): require()
#8 E:\OpenServer\domains\yii\framework\web\CBaseController.php(95): SiteController->renderInternal()
#9 E:\OpenServer\domains\yii\framework\web\CController.php(784): SiteController->renderFile()
#10 E:\OpenServer\domains\yii\protected\controllers\SiteController.php(72): SiteController->render()
#11 E:\OpenServer\domains\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionContact()
#12 E:\OpenServer\domains\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#13 E:\OpenServer\domains\yii\framework\web\CController.php(286): SiteController->runAction()
#14 E:\OpenServer\domains\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#15 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): SiteController->run()
#16 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#17 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#18 E:\OpenServer\domains\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/Yii//index.php?r=site/contact
in E:\OpenServer\domains\yii\protected\views\layouts\main.php (51)
in E:\OpenServer\domains\yii\protected\views\layouts\column1.php (6)
in E:\OpenServer\domains\yii\protected\controllers\SiteController.php (72)
2015/05/18 18:46:32 [error] [exception.CException] exception 'CException' with message 'Контроллер CatController не может найти представление "site".' in E:\OpenServer\domains\yii\framework\web\CController.php:878
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CController.php(782): CController->renderPartial('site', NULL, true)
#1 E:\OpenServer\domains\yii\protected\controllers\CatController.php(11): CController->render('site')
#2 E:\OpenServer\domains\yii\framework\web\actions\CInlineAction.php(49): CatController->actionIndex()
#3 E:\OpenServer\domains\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 E:\OpenServer\domains\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 E:\OpenServer\domains\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('cat')
#8 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#9 E:\OpenServer\domains\yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=cat
---
2015/05/18 18:46:44 [error] [exception.CException] exception 'CException' with message 'Контроллер CatController не может найти представление "site".' in E:\OpenServer\domains\yii\framework\web\CController.php:878
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CController.php(782): CController->renderPartial('site', NULL, true)
#1 E:\OpenServer\domains\yii\protected\controllers\CatController.php(11): CController->render('site')
#2 E:\OpenServer\domains\yii\framework\web\actions\CInlineAction.php(49): CatController->actionIndex()
#3 E:\OpenServer\domains\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 E:\OpenServer\domains\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 E:\OpenServer\domains\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('cat')
#8 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#9 E:\OpenServer\domains\yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=cat
---
2015/05/18 18:47:02 [error] [exception.CException] exception 'CException' with message 'Контроллер CatController не может найти представление "page".' in E:\OpenServer\domains\yii\framework\web\CController.php:878
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CController.php(782): CController->renderPartial('page', NULL, true)
#1 E:\OpenServer\domains\yii\protected\controllers\CatController.php(11): CController->render('page')
#2 E:\OpenServer\domains\yii\framework\web\actions\CInlineAction.php(49): CatController->actionIndex()
#3 E:\OpenServer\domains\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 E:\OpenServer\domains\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 E:\OpenServer\domains\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('cat')
#8 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#9 E:\OpenServer\domains\yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=cat
---
2015/05/18 18:47:10 [error] [exception.CException] exception 'CException' with message 'Контроллер CatController не может найти представление "index".' in E:\OpenServer\domains\yii\framework\web\CController.php:878
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CController.php(782): CController->renderPartial('index', NULL, true)
#1 E:\OpenServer\domains\yii\protected\controllers\CatController.php(11): CController->render('index')
#2 E:\OpenServer\domains\yii\framework\web\actions\CInlineAction.php(49): CatController->actionIndex()
#3 E:\OpenServer\domains\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 E:\OpenServer\domains\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 E:\OpenServer\domains\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('cat')
#8 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#9 E:\OpenServer\domains\yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=cat
---
2015/05/18 18:47:45 [error] [exception.CException] exception 'CException' with message 'Контроллер CatController не может найти представление "index".' in E:\OpenServer\domains\yii\framework\web\CController.php:878
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CController.php(782): CController->renderPartial('index', NULL, true)
#1 E:\OpenServer\domains\yii\protected\controllers\CatController.php(11): CController->render('index')
#2 E:\OpenServer\domains\yii\framework\web\actions\CInlineAction.php(49): CatController->actionIndex()
#3 E:\OpenServer\domains\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 E:\OpenServer\domains\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 E:\OpenServer\domains\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('cat')
#8 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#9 E:\OpenServer\domains\yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=cat
---
2015/05/18 19:35:26 [error] [php] Object of class Cat could not be converted to string (E:\OpenServer\domains\yii\protected\controllers\CatController.php:9)
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CController.php(286): CatController->runAction()
#1 E:\OpenServer\domains\yii\framework\web\CController.php(265): CatController->runActionWithFilters()
#2 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): CatController->run()
#3 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#5 E:\OpenServer\domains\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=cat
in E:\OpenServer\domains\yii\protected\controllers\CatController.php (9)
in E:\OpenServer\domains\yii\index.php (13)
2015/05/18 22:45:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "content".' in E:\OpenServer\domains\yii\framework\web\CController.php:483
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CController.php(270): CController->missingAction('content')
#1 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): CController->run('content')
#2 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/content')
#3 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 E:\OpenServer\domains\yii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/content
HTTP_REFERER=http://yii/index.php?r=content
---
2015/05/18 22:45:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "content".' in E:\OpenServer\domains\yii\framework\web\CController.php:483
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CController.php(270): CController->missingAction('content')
#1 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): CController->run('content')
#2 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/content')
#3 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 E:\OpenServer\domains\yii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/content
HTTP_REFERER=http://yii/index.php?r=content
---
2015/05/18 22:46:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "content".' in E:\OpenServer\domains\yii\framework\web\CController.php:483
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CController.php(270): CController->missingAction('content')
#1 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): CController->run('content')
#2 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/content')
#3 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 E:\OpenServer\domains\yii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/content
HTTP_REFERER=http://yii/index.php?r=site/index
---
2015/05/18 22:46:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "content".' in E:\OpenServer\domains\yii\framework\web\CController.php:483
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CController.php(270): CController->missingAction('content')
#1 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): CController->run('content')
#2 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/content')
#3 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 E:\OpenServer\domains\yii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/content
HTTP_REFERER=http://yii/index.php?r=site/index
---
2015/05/18 22:46:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "content".' in E:\OpenServer\domains\yii\framework\web\CController.php:483
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CController.php(270): CController->missingAction('content')
#1 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): CController->run('content')
#2 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/content')
#3 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 E:\OpenServer\domains\yii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/content
HTTP_REFERER=http://yii/index.php?r=site/index
---
2015/05/18 22:46:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "content".' in E:\OpenServer\domains\yii\framework\web\CController.php:483
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CController.php(270): CController->missingAction('content')
#1 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): CController->run('content')
#2 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/content')
#3 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 E:\OpenServer\domains\yii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/content
HTTP_REFERER=http://yii/index.php?r=site/index
---
2015/05/18 22:46:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "content".' in E:\OpenServer\domains\yii\framework\web\CController.php:483
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CController.php(270): CController->missingAction('content')
#1 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): CController->run('content')
#2 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/content')
#3 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 E:\OpenServer\domains\yii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/content
HTTP_REFERER=http://yii/index.php?r=site/content
---
2015/05/18 22:46:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "content".' in E:\OpenServer\domains\yii\framework\web\CController.php:483
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CController.php(270): CController->missingAction('content')
#1 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): CController->run('content')
#2 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/content')
#3 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 E:\OpenServer\domains\yii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/content
HTTP_REFERER=http://yii/index.php?r=site/index
---
2015/05/18 22:46:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "content".' in E:\OpenServer\domains\yii\framework\web\CController.php:483
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CController.php(270): CController->missingAction('content')
#1 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): CController->run('content')
#2 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/content')
#3 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 E:\OpenServer\domains\yii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/content
HTTP_REFERER=http://yii/index.php?r=site/index
---
2015/05/18 22:46:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "content".' in E:\OpenServer\domains\yii\framework\web\CController.php:483
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CController.php(270): CController->missingAction('content')
#1 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): CController->run('content')
#2 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/content')
#3 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 E:\OpenServer\domains\yii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/content
HTTP_REFERER=http://yii/index.php
---
2015/05/18 22:46:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "content".' in E:\OpenServer\domains\yii\framework\web\CController.php:483
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CController.php(270): CController->missingAction('content')
#1 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): CController->run('content')
#2 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/content')
#3 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 E:\OpenServer\domains\yii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/content
HTTP_REFERER=http://yii/index.php
---
2015/05/18 22:51:16 [error] [php] Object of class Cat could not be converted to string (E:\OpenServer\domains\yii\protected\views\site\index.php:6)
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CController.php(869): CatController->renderFile()
#1 E:\OpenServer\domains\yii\framework\web\CController.php(782): CatController->renderPartial()
#2 E:\OpenServer\domains\yii\protected\controllers\CatController.php(15): CatController->render()
#3 E:\OpenServer\domains\yii\framework\web\actions\CInlineAction.php(49): CatController->actionIndex()
#4 E:\OpenServer\domains\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 E:\OpenServer\domains\yii\framework\web\CController.php(286): CatController->runAction()
#6 E:\OpenServer\domains\yii\framework\web\CController.php(265): CatController->runActionWithFilters()
#7 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): CatController->run()
#8 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 E:\OpenServer\domains\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=cat
in E:\OpenServer\domains\yii\protected\views\site\index.php (6)
in E:\OpenServer\domains\yii\protected\controllers\CatController.php (15)
in E:\OpenServer\domains\yii\index.php (13)
2015/05/18 22:51:26 [error] [php] Object of class Cat could not be converted to string (E:\OpenServer\domains\yii\protected\views\site\index.php:6)
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CController.php(869): CatController->renderFile()
#1 E:\OpenServer\domains\yii\framework\web\CController.php(782): CatController->renderPartial()
#2 E:\OpenServer\domains\yii\protected\controllers\CatController.php(15): CatController->render()
#3 E:\OpenServer\domains\yii\framework\web\actions\CInlineAction.php(49): CatController->actionIndex()
#4 E:\OpenServer\domains\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 E:\OpenServer\domains\yii\framework\web\CController.php(286): CatController->runAction()
#6 E:\OpenServer\domains\yii\framework\web\CController.php(265): CatController->runActionWithFilters()
#7 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): CatController->run()
#8 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 E:\OpenServer\domains\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=cat
in E:\OpenServer\domains\yii\protected\views\site\index.php (6)
in E:\OpenServer\domains\yii\protected\controllers\CatController.php (15)
in E:\OpenServer\domains\yii\index.php (13)
2015/05/18 22:51:46 [error] [php] Object of class Cat could not be converted to string (E:\OpenServer\domains\yii\protected\controllers\CatController.php:13)
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CController.php(286): CatController->runAction()
#1 E:\OpenServer\domains\yii\framework\web\CController.php(265): CatController->runActionWithFilters()
#2 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): CatController->run()
#3 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#5 E:\OpenServer\domains\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=cat
in E:\OpenServer\domains\yii\protected\controllers\CatController.php (13)
in E:\OpenServer\domains\yii\index.php (13)
2015/05/19 23:28:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "users".' in E:\OpenServer\domains\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('users')
#1 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 E:\OpenServer\domains\yii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=users
---
2015/05/19 23:30:03 [error] [system.db.CDbCommand] Не удалось выполнить CDbCommand::fetchAll(): SQLSTATE[42S02]: Base table or view not found: 1146 Table 'db_yii.user' doesn't exist. Выполнявшийся SQL-запрос: SHOW FULL COLUMNS FROM `user`.
in E:\OpenServer\domains\yii\protected\controllers\UserController.php (9)
in E:\OpenServer\domains\yii\index.php (13)
2015/05/19 23:30:03 [error] [exception.CDbException] exception 'CDbException' with message 'Таблица "user", упомянутая в записи active record класса "User", не найдена в базе данных.' in E:\OpenServer\domains\yii\framework\db\ar\CActiveRecord.php:2385
Stack trace:
#0 E:\OpenServer\domains\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 E:\OpenServer\domains\yii\framework\db\ar\CActiveRecord.php(661): CActiveRecord->getMetaData()
#2 E:\OpenServer\domains\yii\framework\db\ar\CActiveRecord.php(1355): CActiveRecord->getTableSchema()
#3 E:\OpenServer\domains\yii\framework\db\ar\CActiveRecord.php(1475): CActiveRecord->query(Object(CDbCriteria), true)
#4 E:\OpenServer\domains\yii\protected\controllers\UserController.php(9): CActiveRecord->findAll()
#5 E:\OpenServer\domains\yii\framework\web\actions\CInlineAction.php(49): UserController->actionIndex()
#6 E:\OpenServer\domains\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 E:\OpenServer\domains\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 E:\OpenServer\domains\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): CController->run('')
#10 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#11 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#12 E:\OpenServer\domains\yii\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/index.php?r=user
---
2015/05/19 23:33:24 [error] [exception.CException] exception 'CException' with message 'Контроллер UserController не может найти представление "users_list".' in E:\OpenServer\domains\yii\framework\web\CController.php:878
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CController.php(782): CController->renderPartial('users_list', Array, true)
#1 E:\OpenServer\domains\yii\protected\controllers\UserController.php(12): CController->render('users_list', Array)
#2 E:\OpenServer\domains\yii\framework\web\actions\CInlineAction.php(49): UserController->actionIndex()
#3 E:\OpenServer\domains\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 E:\OpenServer\domains\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 E:\OpenServer\domains\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#8 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#9 E:\OpenServer\domains\yii\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=user
---
2015/05/19 23:36:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "user".' in E:\OpenServer\domains\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#1 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 E:\OpenServer\domains\yii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=user
---
2015/05/19 23:37:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "user".' in E:\OpenServer\domains\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#1 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 E:\OpenServer\domains\yii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=user
---
2015/05/19 23:38:42 [error] [php] Invalid argument supplied for foreach() (E:\OpenServer\domains\yii\protected\views\user\index.php:2)
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CController.php(869): UserController->renderFile()
#1 E:\OpenServer\domains\yii\framework\web\CController.php(782): UserController->renderPartial()
#2 E:\OpenServer\domains\yii\protected\controllers\UserController.php(6): UserController->render()
#3 E:\OpenServer\domains\yii\framework\web\actions\CInlineAction.php(49): UserController->ActionIndex()
#4 E:\OpenServer\domains\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 E:\OpenServer\domains\yii\framework\web\CController.php(286): UserController->runAction()
#6 E:\OpenServer\domains\yii\framework\web\CController.php(265): UserController->runActionWithFilters()
#7 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): UserController->run()
#8 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 E:\OpenServer\domains\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=user
in E:\OpenServer\domains\yii\protected\views\user\index.php (2)
in E:\OpenServer\domains\yii\protected\controllers\UserController.php (6)
in E:\OpenServer\domains\yii\index.php (13)
2015/05/19 23:39:38 [error] [php] Invalid argument supplied for foreach() (E:\OpenServer\domains\yii\protected\views\user\index.php:2)
Stack trace:
#0 E:\OpenServer\domains\yii\framework\web\CController.php(869): UserController->renderFile()
#1 E:\OpenServer\domains\yii\framework\web\CController.php(782): UserController->renderPartial()
#2 E:\OpenServer\domains\yii\protected\controllers\UserController.php(6): UserController->render()
#3 E:\OpenServer\domains\yii\framework\web\actions\CInlineAction.php(49): UserController->ActionIndex()
#4 E:\OpenServer\domains\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 E:\OpenServer\domains\yii\framework\web\CController.php(286): UserController->runAction()
#6 E:\OpenServer\domains\yii\framework\web\CController.php(265): UserController->runActionWithFilters()
#7 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(282): UserController->run()
#8 E:\OpenServer\domains\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 E:\OpenServer\domains\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 E:\OpenServer\domains\yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=user
in E:\OpenServer\domains\yii\protected\views\user\index.php (2)
in E:\OpenServer\domains\yii\protected\controllers\UserController.php (6)
in E:\OpenServer\domains\yii\index.php (13)
